.github/workflows/govuln.yaml

```diff
@@ -16,4 +16,18 @@ jobs:
       - run: |
           set -euo pipefail
 
-          go install golang.org/x/vuln/cmd/govulncheck@latest && govulncheck ./...
+          go install golang.org/x/vuln/cmd/govulncheck@latest
+
+          # Find all go.mod files recursively, as we have multiple of them within the project
+          gomod_files=$(find . -name go.mod)
+
+          # Loop through each go.mod file path
+          for filepath in $gomod_files; do
```

Comment:
Please refer to https://github.com/etcd-io/etcd/pull/18171#discussion_r1639012673